### PR TITLE
[DP-422] spark - Get rid of compilation warnings

### PR DESCRIPTION
**Pull request by aleksey-suprun**

### CODE REVIEW

**Comment by srgg** [comment]
According to the verification build there is a last warning:

```
[INFO] Checking for multiple versions of scala
[WARNING]  Expected all dependencies to require Scala version: 2.10.5
[WARNING]  com.fasterxml.jackson.module:jackson-module-scala_2.10:2.4.4 requires scala version: 2.10.5
[WARNING]  org.scala-lang:scala-reflect:2.10.4 requires scala version: 2.10.5
[WARNING]  com.twitter:chill_2.10:0.5.0 requires scala version: 2.10.5
[WARNING]  com.typesafe.akka:akka-remote_2.10:2.3.11 requires scala version: 2.10.5
[WARNING]  com.typesafe.akka:akka-actor_2.10:2.3.11 requires scala version: 2.10.5
[WARNING]  com.typesafe.akka:akka-slf4j_2.10:2.3.11 requires scala version: 2.10.5
[WARNING]  org.apache.spark:spark-core_2.10:1.5.2 requires scala version: 2.10.5
[WARNING]  org.json4s:json4s-jackson_2.10:3.2.10 requires scala version: 2.10.5
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.5
[WARNING]  org.json4s:json4s-ast_2.10:3.2.10 requires scala version: 2.10.5
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.0
[WARNING] Multiple versions of scala libraries detected!
[INFO] includes = [**/*.java,**/*.scala,]
[INFO] excludes = []
[INFO] Nothing to compile - all classes are up to date
[INFO] 
```

File: examples/pom.xml [hunk 1]
```diff
@@ -14,9 +14,24 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-swing</artifactId>
```
Comment (srgg) [inline examples/pom.xml]: Do we really need to have scala-swing?

Comment (aleksey-suprun) [inline examples/pom.xml]: I'm afraid that scala-swing is really required. Compilation fails without it

Comment (srgg) [inline examples/pom.xml]: Ok then

**Pull request merged**